libvirt/tests/src/storage/virsh_pool_vol.py

```
@@ -0,0 +1,405 @@
+        if pool_defined:
+            logging.debug("Undefining pool '%s'", pool_name)
+            res = virsh.pool_undefine(pool_name, ignore_status=True, debug=True)
+            if res.exit_status:
+                logging.warning("pool-undefine failed for '%s': %s",
+                                pool_name, res.stderr.strip())
+            else:
+                check_pool_list(expect_present=False)
+
+        if os.path.isdir(pool_target):
+            logging.debug("Removing pool target directory: %s", pool_target)
+            process.run("rm -rf %s" % pool_target,
```

Ruff (0.15.18)

[error] Function call with `shell=True` parameter identified, security issue

(S604)